aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -74,6 +74,13 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.hash.keccak256.base, optional "hash.keccak256.base", 1],
     [.hash.keccak256.per_byte, optional "hash.keccak256.per_byte", 1],
 
+    [.hash.sha2_512.base, optional "hash.sha2_512.base", 1],
+    [.hash.sha2_512.per_byte, optional "hash.sha2_512.per_byte", 1],
+    [.hash.sha3_512.base, optional "hash.sha3_512.base", 1],
+    [.hash.sha3_512.per_byte, optional "hash.sha3_512.per_byte", 1],
+    [.hash.ripemd160.base, optional "hash.ripemd160.base", 1],
+    [.hash.ripemd160.per_byte, optional "hash.ripemd160.per_byte", 1],
+
```

Comment:
let's get the right gas values before landing

Reply:
Done.